src/main/java/de/dennisguse/opentracks/content/data/Track.java

```diff
@@ -144,6 +149,12 @@ public int describeContents() {
             return 0;
         }
 
+        @NonNull
+        @Override
+        public String toString() {
+            throw new RuntimeException("Not supported");
```

Comment:
Isn't that dangerous? Will the debugger crash, when you look at an Id variable?

Reply:
The debugger survives (at leas tin AndroidStudio) and will just to a toString on the exception.
We can remove this afterwards; I just wanted to make sure that we never use toString in old context (aka `"Something" + trackId`) and get unexpected/weird results.